tools/node-modules

```diff
@@ -0,0 +1,185 @@
+#!/bin/sh
+
+set -eu
+
+CACHE_DIR="${XDG_CACHE_HOME-${HOME}/.cache}/cockpit-dev/cache.git"
```

Comment:
Are you suggesting a different cache directory per organisation?  Why?

Reply:
Wouldn't multiple origins in the same cache clone hopelessly collide? It seems much cleaner to have one cache repo per cloned remote repo rather than switching `origin` around every time I set `$GITHUB_BASE`.

Reply:
Right.  Now I understand this.  Of course.  Will fix :)